### PR TITLE
CSS Task, Landing Page

### PR DESCRIPTION
**Pull request by kolawolejohn**
CSS Task-Landing-Page

- I added my signup page
- Redesigned my login page
- changed the landing page name to index.html
- I have a login.html, and signup.html
- My pages can link to each other
- I finally have a single css file named style.css

### CODE REVIEW

File: submissions/kolawoleOgunfowokan/HTML/css/login-style.css [hunk 1]
```diff
@@ -112,11 +114,50 @@ body {
...
+    border: 1px solid red;
+    color: #c74343;
+    font-size: 20px;
+    padding: 10px;
+}
+
+.signup:hover {
+    background-color: magenta;
+    color: #fff;
+
```
Comment (otaruMendez) [inline submissions/kolawoleOgunfowokan/HTML/css/login-style.css]: Please remove space.

File: submissions/kolawoleOgunfowokan/HTML/index.html [hunk 2]
```diff
@@ -1,42 +1,207 @@
...
+    <title>myoja.com</title>
+</head>
+<body>
+    <header class="page-header">
+        <div class="nav-head">
+            <img src="./images/diamond-simple.png" alt="logo" class="logo">
+            <p class="logo-text">myoja.com</p>
+            <div class="njffsaad">
+                <ul class="form-page">
+                    <li class="nav-head-list"><input type="search" name="search" class="search" placeholder="search for products, brand and categories"></li>
```
Comment (otaruMendez) [inline submissions/kolawoleOgunfowokan/HTML/index.html]: This is good but can be better by indenting the elements properly to make it more readable. 
this for example: 
```
<li class="nav-head-list">
    <input type="search" name="search" class="search">
</li>
```
looks much better than what you have currently. what do you think?

Comment (otaruMendez) [reply submissions/kolawoleOgunfowokan/HTML/index.html]: Same applies to other places in your HTML files where you have nested HTML elements on a single line.

Comment (kolawolejohn) [reply submissions/kolawoleOgunfowokan/HTML/index.html]: I have removed spaces where necessary and also did proper indenting, for nested elements.

File: submissions/kolawoleOgunfowokan/HTML/index.html [hunk 3]
```diff
@@ -1,42 +1,311 @@
...
+                        </div>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <section class="products">
+        <h2>Best Products</h2>
+        <div class="contents">
+            <img src="images/HP250.jpg" class="images" alt="HP laptop">
```
Comment (kenny-wealth) [inline submissions/kolawoleOgunfowokan/HTML/index.html:203]: Minor: You could add prices to the products to give the e-commerce feel

**Review by kenny-wealth — APPROVED** [review]

File: submissions/kolawoleOgunfowokan/HTML/css/style.css [hunk 4]
```diff
@@ -32,91 +287,248 @@ body {
...
+    text-align: center;
+}
+
+.signup-form-foot {
+    background-color: #cad3db;
+    width: 100%;
+    padding: 10px;
+    font-weight: bold;
+}
+
```
Comment (goke-epapa) [inline submissions/kolawoleOgunfowokan/HTML/css/style.css:530]: Minor: you can remove the extra space at the end of this file

**Review by goke-epapa — APPROVED** [review]
Well done

**Pull request merged**